web/app.js

```diff
@@ -1722,6 +1722,7 @@ const PDFViewerApplication = {
     eventBus._on("presentationmode", webViewerPresentationMode);
     eventBus._on("print", webViewerPrint);
     eventBus._on("download", webViewerDownload);
+    eventBus._on("save", webViewerSave);
```

Comment:
This requires a corresponding `eventBus._off(...)` line in `PDFViewerApplication.unbindEvents` below.